server/column_scope.cc

```diff
@@ -215,7 +215,8 @@ runIncrementalT(const std::vector<BoundSqlExpression> & exprs,
                     
                 size_t numRows = endOffset - startOffset;
 
-                Val results[exprs.size()];
```

Comment:
Nope, can't do.  It's too slow and this is very performance sensitive.  Come and see me and I'll show you how to address.